components/script/webdriver_handlers.rs

```diff
@@ -382,10 +382,7 @@ fn get_element_in_view_center_point(element: &Element) -> Option<Point2D<i64>> {
         .map(DomRoot::upcast::<Element>)
         .and_then(|body| {
             element
-                .GetClientRects()
-                .iter()
-                // Step 1
```

Comment:
This comment refers to first step in spec:
> Let rectangle be the first [element](https://dom.spec.whatwg.org/#concept-element) of the [DOMRect](https://www.w3.org/TR/geometry-1/#domrect) sequence returned by calling [getClientRects](https://www.w3.org/TR/cssom-view-1/#dom-element-getclientrects)() on [element](https://dom.spec.whatwg.org/#concept-element).

so it should be kept.

Reply:
Thank you @sagudev .
Your response helps a lot.

I'm working on it now.